elixir/apps/domain/test/domain/events/replication_connection_test.exs

```diff
@@ -18,9 +18,11 @@ defmodule Domain.Events.ReplicationConnectionTest do
 
   # Used to test live connection
   setup_all do
-    {connection_opts, config} =
+    {_enabled, config} =
       Application.fetch_env!(:domain, Domain.Events.ReplicationConnection)
-      |> Keyword.pop(:connection_opts)
+      |> Keyword.pop(:enabled)
```

Comment:
Nothing wrong here, just curious for my own knowledge why the `:enable` key is being popped if it's not being used?

Reply:
So that we can pass the rest of the config into `struct()` for ReplicationConnection.

Reply:
As an example:
```
iex(2)> defmodule Foo do
...(2)> defstruct name: "Bar"
...(2)> end
{:module, Foo,
 <<70, 79, 82, 49, 0, 0, 8, 220, 66, 69, 65, 77, 65, 116, 85, 56, 0, 0, 0, 246,
   0, 0, 0, 22, 10, 69, 108, 105, 120, 105, 114, 46, 70, 111, 111, 8, 95, 95,
   105, 110, 102, 111, 95, 95, 10, 97, 116, ...>>, %Foo{name: "Bar"}}
iex(3)> struct(Foo, %{name: "Baz", age: 100})
%Foo{name: "Baz"}
```

Reply:
Ah! I had tested and thought otherwise. Let me update this.